packages/mongodb-rag-core/src/MongoDbVerifiedAnswerStore.ts

```diff
@@ -0,0 +1,68 @@
+import { DatabaseConnection } from "./DatabaseConnection";
+import { VerifiedAnswer, VerifiedAnswerStore } from "./VerifiedAnswer";
+import { FindNearestNeighborsOptions, WithScore } from "./VectorStore";
+import {
+  MakeMongoDbDatabaseConnectionParams,
+  makeMongoDbDatabaseConnection,
+} from "./MongoDbDatabaseConnection";
+
+export function makeMongoDbVerifiedAnswerStore({
+  connectionUri,
+  databaseName,
+}: MakeMongoDbDatabaseConnectionParams): VerifiedAnswerStore &
+  DatabaseConnection {
+  const { db, drop, close } = makeMongoDbDatabaseConnection({
+    connectionUri,
+    databaseName,
+  });
+  const collection = db.collection<VerifiedAnswer>("verified_answers");
```

Comment:
can you make the collection name an optional passable param w/ this as the default name?

i think we should move to that as the default pattern for these stores. we can add the same logic to the other stores as well.